company-profiles/uplers.md

```diff
@@ -0,0 +1,29 @@
+# Uplers
+
+## Company blurb
+
+Uplers is on a mission to provide companies all over the globe with the best remote Indian talent to meet their hiring needs.
```

Comment:
Although you may be on a ` mission to provide companies all over the globe with the best remote Indian talent to meet their hiring needs.`, this doesnt say what the company does. Could you add some extra detail?